app/models/manageiq/providers/kubevirt/inventory/collections.rb

```diff
@@ -36,9 +36,12 @@ def ems_clusters(extra_attributes = {})
     def hardwares(extra_attributes = {})
       attributes = {
         :model_class                 => ::Hardware,
-        :inventory_object_attributes => [
-          :memory_mb,
-        ]
+        :inventory_object_attributes => %i(
+          cpu_cores_per_socket
```

Comment:
Do we already support cpu/cores ? if so, we should expose it in the vm_provision dialog.

Reply:
With this PR we will so once it is merged we can add it to the dialog.